### ISSUE TITLE
can't modify frozen Array when integrate with Rails

### ISSUE THREAD

**Issue by namtx** [bigcommerce/gruf-prometheus#4]
I'm using Rails 6.0.3 for my app, 
when integrating your gem to my app, I faced with this error message:

```
[bigcommerce-prometheus][unknown] Failed to start web instrumentation - can't modify frozen Array - /Users/namtx/.rvm/gems/ruby-2.6.6@backend-fpx/gems/actionpack-6.0.3/lib/action_dispatch/middleware/stack.rb:92:in `unshift'
/Users/namtx/.rvm/gems/ruby-2.6.6@backend-fpx/gems/actionpack-6.0.3/lib/action_dispatch/middleware/stack.rb:92:in `unshift'
/Users/namtx/.rvm/gems/ruby-2.6.6@backend-fpx/gems/bc-prometheus-ruby-0.2.3/lib/bigcommerce/prometheus/instrumentors/web.rb:86:in `setup_middleware'
/Users/namtx/.rvm/gems/ruby-2.6.6@backend-fpx/gems/bc-prometheus-ruby-0.2.3/lib/bigcommerce/prometheus/instrumentors/web.rb:49:in `start'
/Users/namtx/.rvm/gems/ruby-2.6.6@backend-fpx/gems/bc-prometheus-ruby-0.2.3/lib/bigcommerce/prometheus/integrations/railtie.rb:26:in `block in <class:Railtie>'
Loading development environment (Rails 6.0.3)
```
It seems `gruf-prometheus` try to modify the app middleware which is frozen.

**Comment by namtx** [comment]
This one would be fixed by https://github.com/bigcommerce/gruf-prometheus/pull/5

@splittingred Could you help me to check it?

**Comment by splittingred** [comment]
Hi @namtx - 1.3.0 has been published; can you test against that and confirm this is fixed?

**Comment by namtx** [comment]
it is fixed by 1.3.0
Thank you for responding.
